bindata/assets/alerts/cpu-utilization.yaml

```diff
@@ -1,71 +0,0 @@
-apiVersion: monitoring.coreos.com/v1
-kind: PrometheusRule
-metadata:
-  name: cpu-utilization
-  namespace: openshift-kube-apiserver
```

Comment:
To avoid leaking this PrometheusRule resource in clusters updated from the old situation to the new situation, you should keep this manifest around (at least for 4.11, you can drop it in 4.12) with [the `release.openshift.io/delete: "true"` annotation][1].

[1]: https://github.com/openshift/enhancements/blob/master/enhancements/update/object-removal-manifest-annotation.md

Reply:
Oh wow, I wasn't aware of that annotation, thank you for sharing. I used to write some logic in operators to clean up the manifests.